packages/react-charting/UnitTests/AreaChartUT.test.tsx

```diff
@@ -637,8 +643,11 @@ runTest('_getAriaLabel', () => {
     container => {
       const points = getById(container, /circle/i);
       expect(points).toHaveLength(1);
-      expect(points[0].getAttribute('aria-label')).toEqual('Jan 1, 2022. Legend 1, 10.');
+      expect(points[0].getAttribute('aria-label')).toMatchSnapshot();
     },
+    undefined,
+    undefined,
+    !isTimezoneSet(Timezone.UTC),
```

Comment:
what are you doing here.

Reply:
We are planning to execute these test cases only in UTC time zone and we are passing UTC time sone as input to this function.